### ISSUE TITLE
A seperate page for each post

### ISSUE THREAD

**Issue by Joshitha18** [harsh-9in/Blogger#13]
Each title of post on home page directs to a new page containing details of that post

**Comment by Joshitha18** [comment]
Can I work on it?

**Comment by harsh-9in** [comment]
the title of the post is not redirecting to any page till now. I want you to add the feature of redirecting it to a detailed summary page of the blog.
Am I clear to you? @Joshitha18

**Comment by harsh-9in** [comment]
you have to create a page that contains a detail summary about the blog and connect it through the title.

**Comment by harsh-9in** [comment]
Do you want to work on this issue? @Joshitha18

**Comment by Joshitha18** [comment]
> Do you want to work on this issue? @Joshitha18

Yes

**Comment by harsh-9in** [comment]
That's Great I am assigning this issue to @Joshitha18. Go ahead :)

**Comment by Joshitha18** [comment]
> the title of the post is not redirecting to any page till now. I want you to add the feature of redirecting it to a detailed summary page of the blog.
> Am I clear to you? @Joshitha18
Yeah, the redirected page will contain all the information regarding that post

**Comment by harsh-9in** [comment]
> > the title of the post is not redirecting to any page till now. I want you to add the feature of redirecting it to a detailed summary page of the blog.
> > Am I clear to you? @Joshitha18
> 
> Yeah, the redirected page will contain all the information regarding that post

Yes, it will contain a detailed description of that for now.

**Comment by subhangi2731** [comment]
can i work on this? @harsh-9in

**Comment by sakshikhachane** [comment]
@harsh-9in Can you assign me this issue?

**Comment by harsh-9in** [comment]
assigned to @sakshikhachane

**Comment by sakshikhachane** [comment]
@harsh-9in Is this issue applicable for DWOC?

**Comment by harsh-9in** [comment]
yess
